tests/configSanitized.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>require.js: sanitizeConfig Test</title>
+    <script type="text/javascript" src="../require.js"></script>
+    <script type="text/javascript" src="doh/runner.js"></script>
+    <script type="text/javascript" src="doh/_browserRunner.js"></script>
+    <script>
+        function sanitizeConfigMock(config) {
```

Comment:
If the sanitizeConfig is changed, it does not reflect directly in this mock. I don't know the technical limitations, but it would be much better to ensure a change would be reflected directly on the test.

Reply:
I first started by trying that ( `require.config(inputConfig);` ) but in this case the result is a function from which I cannot seem to extract the necessary information for the adequate asserts.
![image](https://github.com/OutSystems/requirejs/assets/81638586/7e228937-2a43-4895-ac42-3fad37d10adb)
I then tried to export or make the function `santitizeConfig` globally accessible, but this was leading to some other tests falling and seemed like I was doing a big 🔨 , besides, I don't feel like the function should be accessible outside the `require.js` file.
Lastly, I went for this alternative, although I understand and agree with your comment.